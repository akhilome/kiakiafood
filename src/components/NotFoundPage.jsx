import React from 'react';
import { Link } from 'react-router-dom';

const NotFoundPage = () => (
  <div>
    Four-oh-Four
    <br />
    <Link to="/">Go Home</Link>
  </div>
);

export default NotFoundPage;
