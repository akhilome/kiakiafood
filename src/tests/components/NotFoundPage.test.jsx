import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

describe('NotFoundPage', () => {
  it('should render NotFoundPage correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
