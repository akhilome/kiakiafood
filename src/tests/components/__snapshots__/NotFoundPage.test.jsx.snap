// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render NotFoundPage correctly 1`] = `
<div>
  Four-oh-Four
  <br />
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
